test/active_model_serializers/test/serializer_test.rb

```diff
@@ -17,6 +23,11 @@ def render_some_text
 
       tests MyController
 
+      def test_supports_specifying_serializers_with_a_multi_serializer_class
+        get :render_multi_using_serializer
+        assert_serializer ProfileSerializer
```

Comment:
Perhaps what you want is `assert_each_serializer`?  I think it is correct to say the serializer is `ActiveModel::Serializer::CollectionSerializer`. In any case, you don't need any assertion when you provide the serializer explicitly.